tests/queries/test_qs_combinators.py

```diff
@@ -42,6 +42,30 @@ def test_union_distinct(self):
         self.assertEqual(len(list(qs1.union(qs2, all=True))), 20)
         self.assertEqual(len(list(qs1.union(qs2))), 10)
 
+    @skipUnlessDBFeature('supports_select_intersection')
+    def test_intersection_with_empty_qs(self):
+        qs1 = Number.objects.all()
+        qs2 = Number.objects.none()
+        self.assertEqual(len(list(qs1.intersection(qs2))), 0)
```

Comment:
In the original review of this file, I missed the fact (unless I'm missing something now) that there's no need to call `list()` inside of `len()`.